components/HospitalCard.js

```diff
@@ -1,18 +1,39 @@
 import { Heading, Box, HStack, VStack, Text, Flex } from '@chakra-ui/react'
 import { PhoneIcon } from '@chakra-ui/icons'
+import { point, distance } from '@turf/turf'
 
 import { getRelativeLastUpdatedTime } from '@/utils/HospitalHelper'
 
 function generateGoogleMapsLink(name) {
   return `https://www.google.com/maps/search/${name}`
 }
 
+function calculateDistance(from, to, unit) {
+  const satuan = unit ? unit : 'kilometers'
```

Comment:
maybe can be written like this: `const satuan = unit ?? 'kilometers'`

Reply:
Oh thanks @sozonome :+1: